docker-bake.hcl

```diff
@@ -112,7 +112,10 @@ variable "AWS_S3_BUCKET" {
   default = ""
 }
 variable "AWS_S3_CONFIG" {
-  default = "_website-config-docs-stage.json"
+  default = ""
```

Comment:
Does the Dockerfile need some default (or a check if it's empty?) or no concerns there?

Reply:
As we always sets the value from our ci I think it's fine. This var will be removed in a follow-up anyway and will only use `s3-config.json`.

Reply:
sgtm, thx!